lib/tracer.js

```diff
@@ -0,0 +1,19 @@
+// Don't require agent until needed, early require causes its configuration to
+// be loaded from the CWD, and the CWD isn't the directory of the application
+// until run has had a chance to chdir() into it.
+var assert = require('assert');
+
+module.exports = tracer;
+
+var TRACER = null;
+
+function tracer(options) {
+  // XXX sl-run.js sets the agent logger, and the quiet option, that won't
```

Comment:
cut-n-paste comment, it doesn't apply to tracer, which has no logger or quiet option.